cpp/Platform.Equality.Tests/Platform.Equality.Benchmark.cpp

```diff
@@ -0,0 +1,126 @@
+#include <Platform.Equality.h>
+#include <benchmark/benchmark.h>
+
+#include <ctime>
+#include <iostream>
+#include <span>
+#include <any>
+
+namespace Platform::Equality::Benchmark
+{
+    namespace Any
+    {
+        static void AnyCompare(benchmark::State& state)
+        {
+            srand(time(nullptr));
+
+            for (auto _ : state)
+            {
+                auto a = rand();
+                auto b = rand();
+
+                bool result;
+                benchmark::DoNotOptimize(result = std::any(a) == std::any(b));
+            }
+        }
+
+        BENCHMARK(AnyCompare);
+
+        static void NormalCompare(benchmark::State& state)
+        {
+            srand(time(nullptr));
+
+            for (auto _ : state)
+            {
+                auto a = rand();
+                auto b = rand();
+
+                bool result;
+                benchmark::DoNotOptimize(result = a == b);
+            }
+        }
+
+        BENCHMARK(NormalCompare);
+    }
+
+    namespace Range
+    {
+        namespace Internal
+        {
+            inline constexpr int test_size = 10000;
```

Comment:
constexpr implicitly implies `const` and `inline`, so only constexpr can be left
```cpp
constexpr int test_size = 10000;
```